presto-docs/src/main/sphinx/release/release-0.122.rst

```diff
@@ -9,3 +9,10 @@ General Changes
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
 * * Fix ``NoSuchElementException`` when cross join is used inside ``IN`` query.
+* Fixed ``GROUP BY`` to suport columns containing maps of structural types.
+* The web interface now displays a lock icon next to authenticated users.
+* The ``min_by`` and ``max_by`` aggregations now take a third parameter ``n``, 
```

Comment:
"The ... aggregations now have an additional form that return multiple values."

Use function link syntax for the functions